test/replcompletions.jl

```diff
@@ -161,7 +161,7 @@ c,r,res = test_complete(s)
 
 s = "\\a"
 c, r, res = test_complete(s)
-"\\alpha" in c
+("\\alpha" in c)
```

Comment:
Yes, since previously the string was being interpreted as a docstring. Not sure what it was documenting but there you go.